lib/router-config/src/traffic_shaping.rs

```diff
@@ -51,3 +54,17 @@ fn default_dedupe_enabled() -> bool {
 fn default_dedupe_fingerprint_headers() -> Vec<String> {
     vec!["authorization".to_string()]
```

Comment:
Should this also include `Cookie`? It's fairly common (and more secure) to use it instead of `Authorization` header